test/e2e/source_kafka_test.go

```diff
@@ -0,0 +1,122 @@
+// Copyright 2020 The Knative Authors
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+//     http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build e2e
+// +build !serving
+
+package e2e
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	testcommon "github.com/maximilien/kn-source-pkg/test/e2e"
+	"gotest.tools/v3/assert"
+	"knative.dev/client/lib/test"
+	"knative.dev/client/pkg/util"
+)
+
+const (
+	kafkaBootstrapUrl     = "my-cluster-kafka-bootstrap.kafka.svc:9092"
+	kafkaClusterName      = "my-cluster"
+	kafkaClusterNamespace = "kafka"
+	kafkaTopic            = "test-topic"
+)
+
+type e2eTest struct {
+	it *testcommon.E2ETest
+}
+
+func newE2ETest(t *testing.T) *e2eTest {
+	currentDir, err := os.Getwd()
+	if err != nil {
+		return nil
+	}
+
+	it, err := testcommon.NewE2ETest("kn-source-kafka", filepath.Join(currentDir, "../.."), false)
```

Comment:
I'm a bit confused: Do we want to test the plugin in isolation or when inlined ?

Reply:
Well, it's inlined, but that's inherited from original test. The install setup is disabled though, see [here](https://github.com/openshift/knative-client/pull/696/files#diff-7f2991afe73983ad008d523abe23e7b283269926f48e8926927586daadafe36aR71-R72).

Reply:
Ah, I see. thx! Maybe add a comment to the commented-out code to explain the purpose ? Also using a file path here is kind of confusing, too. (so maybe also add a comment here ?) 

Otherwise PR looks good to me.